rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.desired_nodes/10_basic.yml

```diff
@@ -774,3 +767,135 @@ teardown:
             - { settings: { "node.external_id": "instance-000187"}, processors: 64, memory: "1b", storage: "1b", node_version: null }
   - match: { status: 400 }
   - match: { error.type: x_content_parse_exception }
+---
+"Test update desired nodes using legacy processors field":
+  - skip:
+      reason: "contains is a newly added assertion"
+      features: contains
```

Comment:
It looks like we are not using `contains` here (as well as the rest of the new tests)?

Reply:
Copy-paste... 🤦